Panels/tests/Header-specs.js

```diff
@@ -1,42 +1,39 @@
-import {mount} from 'enzyme';
+import '@testing-library/jest-dom';
+import {render, screen} from '@testing-library/react';
+
 import Header from '../Header';
-import css from '../Header.module.less';
 
 describe('Header Specs', () => {
-
 	test('should render with title text without changing case', () => {
 		let msg = 'cRaZy-cased super Header';
-
-		const header = mount(
+		render(
```

Comment:
You can put the render on a single line. Same below. The last test is ok with multiple lines.